packages/node_modules/@webex/webex-core/test/integration/spec/services/services.js

```diff
@@ -767,6 +767,31 @@ describe('webex-core', () => {
             assert(true);
           });
       });
+
+      it('updates limited catalog and calls _fetchNewServiceHostmap with forceRefresh = true', (done) => {
+        const forceRefresh = true;
+
+        catalog.updateServiceUrls = sinon.stub().returns({}); // returns `this`
```

Comment:
Instead of stubbing it out, can you spy on it? That way we can see if this method gets called with the correct parameters and insure we get the correct response from the server.

Reply:
Done!